core/controllers/admin_test.py

```diff
@@ -91,34 +91,34 @@ def test_change_configuration_property(self):
         response_config_properties = response_dict['config_properties']
         self.assertDictContainsSubset({
             'value': self.UNICODE_TEST_STRING,
-        }, response_config_properties[pages.MODERATOR_REQUEST_FORUM_URL.name])
+        }, response_config_properties[base.BEFORE_END_HEAD_TAG_HOOK.name])
 
         self.logout()
 
     def test_change_about_page_config_property(self):
         """Test that the correct variables show up on the about page."""
-        new_contact_email = 'test2@example.com'
+        new_config_value = 'abcd'
 
         # Navigate to the about page. The site name is not set.
         response = self.testapp.get('/about')
-        self.assertIn('CONTACT_EMAIL_ADDRESS', response.body)
-        self.assertNotIn(new_contact_email, response.body)
+        self.assertIn('BEFORE_END_HEAD_TAG_HOOK', response.body)
```

Comment:
The default value for this property is empty `''` (Refer: line 78 of this file). Hence, this assertion fails.